### ISSUE TITLE
Reconsider whether we want to continue using an analytics service

### ISSUE THREAD

**Issue by WorldLanguages** [ScratchAddons/website-v2#320]
I think we often forget that we have https://analytics.scratchaddons.com, so I don't see the point of keeping it.

We should really consider having no website analytics at all, or find out if we have reached any meaningful conclusions after all these months - probably not.

**Comment by Secret-chest** [comment]
The `/welcome` and `/farewell` pages could be pretty useful.

**Comment by BroJac5246** [comment]
How is this a bug?

**Comment by WorldLanguages** [comment]
> The `/welcome` and `/farewell` pages could be pretty useful.

I think extension stores provide similar analytics, but they don't easily let you make them public.
And serious question: will our decisions depend on how-many/when users are installing or removing the extension? I don't really think so

**Comment by WorldLanguages** [comment]
Asking for more opinions: @mxmou @jeffalo @DNin01 @lisa-wolfgang @Hans5958

**Comment by Samq64** [comment]
I don't think it's really necessary. It could be useful but the only thing I didn't realize was  how many Chromebook users we have. For the welcome and farewell pages, there are other ways to see the number of activitie users, although I'm not sure if any of them are public besides the count om the main site page.

**Comment by Secret-chest** [comment]
The welcome or farewell pages could also tell us how many users we gained in a time period.

**Comment by Hans5958** [comment]
I'm a little bit neutral on this one. While I don't refer to this page that often (partly because I don't save it as a bookmark), I would find it useful from time to time.

**Comment by BroJac5246** [comment]
Is there a point to getting rid of it? Are we self-hosting or paying for it? I think that if there's no point to getting rid of it then we should keep it, but if there is a point or we're paying a meaningful amount of money then we should get rid of it because it's not worth it.

**Comment by WorldLanguages** [comment]
> Is there a point to getting rid of it?

I think the main reason to get rid of website analytics is having a more concise privacy policy. The less things we have to mention in the privacy policy, the better. Plausible is better than Google Analytics (which we used at some point) but it's even better not to use any analytics service if we're not finding it useful.

> Are we self-hosting or paying for it?

It has been (and is) hosted by Jeffalo for a while. At some point in the future, I might have to pay for a server (but many other ideas require a server, so I wouldn't think this is a problem).

**Comment by WorldLanguages** [comment]
> The only thing I didn't realize was how many Chromebook users we have.

> The welcome or farewell pages could also tell us how many users we gained in a time period.

Both of these are already part of Chrome Web Store's analytics:

![image](https://user-images.githubusercontent.com/17484114/235365543-b6212c62-8095-47f7-ad5e-c641f2373e5f.png)

**Comment by WorldLanguages** [comment]
Maybe I could find a way to make Chrome Web Store analytics public, and then we can reconsider to stop using Plausible.

**Comment by WorldLanguages** [comment]
Would anyone be **completely against** removing Plausible analytics even if Chrome Web Store analytics were made public and visible through a dashboard similar to [analytics.scratchaddons.com](https://analytics.scratchaddons.com)?

I believe it's a better replacement, and it would also mean that the data collection is in charge of Google and not us. Of course, these two aren't mutually exclusive, but it would make our privacy practices even easier to understand as there won't be any analytics (collected directly by us) at all.

**Comment by DNin01** [comment]
> Maybe I could find a way to make Chrome Web Store analytics public, and then we can reconsider to stop using Plausible.

I suppose that's not a bad idea. We've never really needed anything more to function.

**Comment by Secret-chest** [comment]
Can we add optional anonymous telemetry to the extension? Useful for finding out most used addons if you don't use the feedback form

**Comment by Hans5958** [comment]
> Maybe I could find a way to make Chrome Web Store analytics public, and then we can reconsider to stop using Plausible.

I have to tell you that, personally, I use the analytics not as an indicator of SA users, but as an indicator of the website's users. It is nice that I knew what pages are high-traffic or not thanks to the analytics.

**Comment by WorldLanguages** [comment]
> I have to tell you that, personally, I use the analytics not as an indicator of SA users, but as an indicator of the website's users. It is nice that I knew what pages are high-traffic or not thanks to the analytics.

Good point. Let's postpone this discussion.
Related: I also have access to Google Search Console data that I would ideally like to make public too. But it wouldn't be a direct replacement to know which pages are high-traffic.

**Comment by WorldLanguages** [comment]
> Can we add optional anonymous telemetry to the extension? Useful for finding out most used addons if you don't use the feedback form

That is off-topic (I would say it's even the wrong repository) and it depends on how we'd implement the "optional telemetry". If you have to enable telemetry, the resulting data will be biased in favor of addons enabled by users that looked deeply enough into the settings in order to find the telemetry switch. We end up with more or less the same dataset than using the feedback form but we'd need to change the privacy policy and justify the telemetry. Can't say it would be worth it without discussing the details.

**Comment by Secret-chest** [comment]
It would offer a choice of telemetry on first launch.

**Comment by DNin01** [comment]
> It would offer a choice of telemetry on first launch.

That still wouldn't solve the problem of people opting out and skewing the data.

I learned somewhere that you can "un-bias" statistics by creating a subset of the data you have collected by randomly selecting entries from it. This would create a smaller portion of the data that supposedly helps "un-bias" the data, so that could help with this.

**Comment by BroJac5246** [comment]
> > It would offer a choice of telemetry on first launch.
> 
> That still wouldn't solve the problem of people opting out and skewing the data.
> 
> I learned somewhere that you can "un-bias" statistics by creating a subset of the data you have collected by randomly selecting entries from it. This would create a smaller portion of the data that supposedly helps "un-bias" the data, so that could help with this.

How does that work? All you're doing is taking a subset of the biased data, so you'll just randomly select a biased subset. We could figure out how many people opted out, but I don't think that would actually help.

Ultimately, will bias play a major role? As long as every user is prompted to pick their telemetry preferences (new users upon first download; existing users upon updating), then the bias will be gone because every user is asked if they're okay with it. We can (and should) also provide instructions on how to access telemetry settings from the settings page. As for the privacy policy, all we need to do is add a section. And when we ask the user if they're okay with telemetry, we should be very clear that it is anonymous and we only collect browser, operating system, and enabled settings (I would assume).

**Comment by Secret-chest** [comment]
Yes, anonymous browser version, OS, SA version, possibly CPU/GPU info, enabled addons...

**Comment by DNin01** [comment]
You know, sometimes analytics can help in unexpected ways. Google even kept track of how many misspelled search queries were submitted in their search engine so they could mention that statistic in their YouTube video about dealing with spelling mistakes.

The website analytics could still be useful to show how much traffic we receive on the website. We may have situations where we want to show our user base in a few different ways. Either way, I'd also like to consider doing this in the extension through optional telemetry and maybe even surveys to help improve it.

Anybody have any other ideas for things we could do with analytics?

**Comment by WorldLanguages** [comment]
Closed by #548
